documentation/API_documentation/QualityByDesign_User_Story.md

```diff
@@ -0,0 +1,15 @@
+# Quality-By-Design (QbD) User Story
+
+| **Item**                        | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| ------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **_Summary_**                   | As an application developer, I want to be able to share network KPI requirements with a Communication Service Provider (CSP) and stream these KPIs for real-time monitoring. This allows the CSP to track network performance and ensure it aligns with the specified requirements. If any deviations occur, the CSP can proactively suggest or implement corrective measures to maintain optimal performance.                                                                                                                      |
+| **_Roles, Actor(s) and scope_** | **Roles:** API Consumer:Developer<br><br>**Actors:** Application service providers, hyperscalers, application developers.<br><br>**Scope:**<ul><li>Create QbD Session</li><li>Send Network KPIs</li><li>Get QbD score</li><li>Get RCA and recommended corrective actions</li><li>End Session</li></ul>                                                                                                                                                                                                                              |
+| **_Pre-conditions_**            | The preconditions are:<ol><li>Customer:Developer is eligible to use the QbD API, i.e. the Customer has successfully subscribed to the QbD API on the platform of an API Provider.</li><li>The customer application has requested and received an access token for the required scopes of the API.</li><li>The customer application determines which application profile it will be using for the QbD session.</li></ol>                                                                                                             |
```

Comment:
>The customer application determines which application profile it will be using for the QbD session.

Or is the prerequisite rather that the application developer provisions an application profile first for that application?

Reply:
good point, will update